src/neo/SmartContract/ApplicationEngine.Native.cs

```diff
@@ -19,6 +19,7 @@ protected internal void DeployNativeContracts()
                 {
                     Id = contract.Id,
                     Script = contract.Script,
+                    Hash = contract.Hash, // Use the native hash
```

Comment:
Can we use the same way to compute the native contracts' hash?

Reply:
We can apply it in https://github.com/neo-project/neo/pull/2056

Reply:
Fixed.